Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+	config.vm.box = "centos/7"
+	config.vm.synced_folder ".", "/vagrant", disabled: true
+	config.vm.synced_folder "./samples", "/converge_samples"
+	config.vm.provision :converge do |cvg|
+		cvg.bikeshed = [
+			"/converge_samples/basic.hcl"
+		]
+		cvg.install = true
+	end
+end
```

Comment:
this is out of date with the most recent version of the Vagrant plugin.

```
There are errors in the configuration of this machine. Please fix
the following errors and try again:

vagrant-converge:
* The following settings shouldn't exist: bikeshed
* No value provided for hcl


There are errors in the configuration of this machine. Please fix
the following errors and try again:

vagrant-converge:
* The following settings shouldn't exist: bikeshed
* No value provided for hcl
```

Also, effective testing using vagrant is blocked by #356. I wonder if we should just leave this part out of the PR until we solve those problems.